client/src/Movies/SavedList.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 export default class SavedList extends Component {
   constructor(props) {
```

Comment:
Feel free to clean up your code by removing constructors that are not in use - aka `useless constructors`.